[2025-02-28T04:16:37.736+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-28T04:16:37.783+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-27T00:00:00+00:00 [queued]>
[2025-02-28T04:16:37.811+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-27T00:00:00+00:00 [queued]>
[2025-02-28T04:16:37.812+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-28T04:16:37.841+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-02-27 00:00:00+00:00
[2025-02-28T04:16:37.858+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=415310) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-28T04:16:37.859+0000] {standard_task_runner.py:72} INFO - Started process 415344 to run task
[2025-02-28T04:16:37.859+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2025-02-27T00:00:00+00:00', '--job-id', '592', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp1sslg5ea']
[2025-02-28T04:16:37.861+0000] {standard_task_runner.py:105} INFO - Job 592: Subtask second_task
[2025-02-28T04:16:37.950+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-27T00:00:00+00:00 [running]> on host 25887b48f6fb
[2025-02-28T04:16:38.114+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-27T00:00:00+00:00'
[2025-02-28T04:16:38.116+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-28T04:16:38.116+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-28T04:16:38.117+0000] {logging_mixin.py:190} INFO - Current task name:second_task state:running start_date:2025-02-28 04:16:37.784776+00:00
[2025-02-28T04:16:38.117+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag_v1 and current dag run status:running
[2025-02-28T04:16:38.118+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-28T04:16:38.119+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-28T04:16:38.121+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-02-28T04:16:38.144+0000] {subprocess.py:99} INFO - Output:
[2025-02-28T04:16:38.148+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1!
[2025-02-28T04:16:38.148+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-28T04:16:38.205+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-28T04:16:38.206+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, run_id=scheduled__2025-02-27T00:00:00+00:00, execution_date=20250227T000000, start_date=20250228T041637, end_date=20250228T041638
[2025-02-28T04:16:38.252+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-28T04:16:38.252+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-28T04:16:38.253+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag_v1 queued_at:2025-02-28 04:16:33.292207+00:00
[2025-02-28T04:16:38.253+0000] {logging_mixin.py:190} INFO - Task hostname:25887b48f6fb operator:BashOperator
[2025-02-28T04:16:38.276+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-28T04:16:38.308+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-28T04:16:38.311+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-04T05:11:19.320+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-04T05:11:19.342+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-27T00:00:00+00:00 [queued]>
[2025-03-04T05:11:19.350+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-27T00:00:00+00:00 [queued]>
[2025-03-04T05:11:19.350+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-04T05:11:19.370+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-02-27 00:00:00+00:00
[2025-03-04T05:11:19.383+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=238) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-04T05:11:19.382+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2025-02-27T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpegrcfczr']
[2025-03-04T05:11:19.384+0000] {standard_task_runner.py:72} INFO - Started process 255 to run task
[2025-03-04T05:11:19.384+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask second_task
[2025-03-04T05:11:19.434+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-27T00:00:00+00:00 [running]> on host e52f68431fc8
[2025-03-04T05:11:19.526+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-27T00:00:00+00:00'
[2025-03-04T05:11:19.527+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-04T05:11:19.543+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-04T05:11:19.544+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-03-04T05:11:19.558+0000] {subprocess.py:99} INFO - Output:
[2025-03-04T05:11:19.560+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1!
[2025-03-04T05:11:19.561+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-04T05:11:19.593+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-04T05:11:19.593+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, run_id=scheduled__2025-02-27T00:00:00+00:00, execution_date=20250227T000000, start_date=20250304T051119, end_date=20250304T051119
[2025-03-04T05:11:19.638+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-04T05:11:19.648+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-07T10:44:08.097+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-27T00:00:00+00:00 [queued]>
[2025-03-07T10:44:08.105+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-27T00:00:00+00:00 [queued]>
[2025-03-07T10:44:08.105+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T10:44:08.117+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2025-02-27 00:00:00+00:00
[2025-03-07T10:44:08.123+0000] {standard_task_runner.py:57} INFO - Started process 352 to run task
[2025-03-07T10:44:08.127+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2025-02-27T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpp0b87ddk']
[2025-03-07T10:44:08.130+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask second_task
[2025-03-07T10:44:08.194+0000] {task_command.py:416} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-27T00:00:00+00:00 [running]> on host 93934f8e361f
[2025-03-07T10:44:08.286+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-27T00:00:00+00:00'
[2025-03-07T10:44:08.287+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-07T10:44:08.287+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-03-07T10:44:08.304+0000] {subprocess.py:86} INFO - Output:
[2025-03-07T10:44:08.306+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2025-03-07T10:44:08.306+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-07T10:44:08.338+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, execution_date=20250227T000000, start_date=20250307T104408, end_date=20250307T104408
[2025-03-07T10:44:08.378+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T10:44:08.496+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
